Middleware/DoctrineFlushMiddleware.php

```diff
@@ -31,6 +31,10 @@ public function __construct(EntityManagerInterface $entityManager)
      */
     public function handle(Envelope $envelope, StackInterface $stack): Envelope
     {
+        if (!empty($envelope->all(DisableFlushStamp::class))) {
```

Comment:
i think this will mess up the `this->messageDepth` value. we should probably do it on line 48 🙂

Reply:
Am I right, that we have to flush when `messageDepth` is `0` if at least one `Envelope` doesn't have the `DisableFlushStamp`? Then I would have to introduce a `private $shouldFlush = false` and set that to true, as soon as an `Envelope` doesn't have the `DisableFlushStamp`.

Reply:
i think i would just look at the `Envelope` at the root level (`messageDepth=0`) 🙂

Reply:
now that i think about it - wouldnt it be more intuitive if there was a `EnableFlushStamp` to enable the middleware? 🙈 i know thats a big bc break and we probably wont do that

Reply:
then we could for example also add an additional `EnableFlushWhenHandledcAsyncStamp` for imports. this way we need to add something like a `disableFlushWhenHandledAsync` flag to this messge

Reply:
@nnatter would not change it for backward compatibility.

Reply:
not using the middleware isnt very easy unfortunately - that is why we need this stamp in the first place 😂

Reply:
but yeah, i am just thinking out loud. dont think it would be a good idea to change this in this PR.

Reply:
@nnatter @alexander-schranz fixed